File: python-2.6.4/skeleton.py
#!/usr/bin/python
# -*- coding: utf-8 -*-
# vi:encoding=utf-8

from TestEnv import TestEnv

te= TestEnv()

te.checkComplainAndAdjustExpected( 0)

te.test( True)

te.checkComplainAndAdjustExpected( 1)

# see also: 

print te.result()

File: php5/skeleton.php
#!/usr/bin/php

<?
 require "test-env.inc";

 $te= new TestEnv();

 $te->checkComplainAndAdjustExpected( 0);

 $te->test( true);

 $te->checkComplainAndAdjustExpected( 1);

 echo $te->result(). "\n";
?>
